src/viser/_gui_api.py

```diff
@@ -867,6 +870,40 @@ def add_button_group(
             )._impl,
         )
 
+    def add_notification(
+        self,
+        title: str,
+        body: str,
+        withCloseButton: bool,
```

Comment:
can we adopt the Python naming conventions? like `with_close_button` instead of `withCloseButon`